app/models/grader_permission.rb

```diff
@@ -0,0 +1,15 @@
+# Contains the grader permissions for a particular grader
+class GraderPermission < ApplicationRecord
```

Comment:
This method can be condensed to:

```
class GraderPermission < ApplicationRecord
  belongs_to :ta, class_name: 'Ta', foreign_key: :user_id
end
```

because:

- a `belongs_to` association requires the associated object to exist
- by setting `class_name: 'Ta'` it will check if the user is a Ta for you